supervisor/misc/scheduler.py

```diff
@@ -40,8 +41,8 @@ def _run_task(self, task_id):
         """Run a scheduled task."""
         data = self._data[task_id]
 
-        if not self.suspend:
-            self.loop.create_task(data[CALL]())
+        if self.sys_core.state == CoreStates.RUNNING:
+            self.sys_loop.create_task(data[CALL]())
```

Comment:
Is the context unspecific here, so we need to specify what loop to use when creating the task, Ie use `loop.create_task` instead of `asyncio.create_task`?

Reply:
It's `sys_create_task`. asyncio.create_task has more overhead which we can avoid anyway with our structure they we created before with python 3.4

Reply:
Ok. In the long run I think it's wise to try to move to the recommended higher level APIs. This is a trend in asyncio.

Reply:
Sure, I wait for executor/callback pool to have a high-level API and we can migrate once all to it. maybe with Python 4 - anyway there are ~53 bigger tasks on ClickUp